arduino-hal/src/clock.rs

```diff
@@ -1,8 +1,10 @@
 pub use avr_hal_generic::clock::*;
 
 pub(crate) mod default {
-    #[cfg(feature = "arduino-uno")]
-    pub type DefaultClock = avr_hal_generic::clock::MHz16;
-    #[cfg(feature = "arduino-leonardo")]
+    #[cfg(any(
+        feature = "arduino-uno",
+        feature = "arduino-leonardo",
+        feature = "arduino-mega"
+    ))]
     pub type DefaultClock = avr_hal_generic::clock::MHz16;
```

Comment:
I know that I've been preaching to use "unified" cfgs whereever possible but I'd make this an exception.  I think it is much easier to see which clock rate associates with which board if they are explicitly separate here.  But YMMV...

Reply:
Splitted.